lib/Wikia/src/Service/User/Preferences/PreferenceModule.php

```diff
@@ -10,68 +11,28 @@
 use Wikia\Persistence\User\Preferences\PreferencePersistenceSwaggerService;
 
 class PreferenceModule implements Module {
-	const SWAGGER_SERVICE_RAMP_USAGE = 0;
+	const PREFERENCE_CACHE_VERSION = 1;
```

Comment:
Hmm.. I hope not, but I think the version of [`MW_USER_VERSION `](https://github.com/Wikia/app/blob/dev/includes/User.php#L40) was `1` when we started, so probably